### PR TITLE
Fixed a couple issues with parent-child controllers.

### PR DESCRIPTION
**Pull request by Brandon-T**
Controllers need the proper function calls to have correct parent child relationship otherwise viewWillAppear and viewWillDisappear won't be called.

Also need to constrain the child controller's view.

### CODE REVIEW

File: BraveRewardsUI/Ads/AdsViewController.swift [hunk 1]
```diff
@@ -297,6 +297,10 @@ extension AdsViewController {
     
     presentingController.addChild(adsViewController)
     presentingController.view.addSubview(adsViewController.view)
+		adsViewController.didMove(toParent: presentingController)
```
Comment (kylehickinson) [inline BraveRewardsUI/Ads/AdsViewController.swift]: Some sort of whitespace issue going on here (using tabs instead of spaces)

Comment (kylehickinson) [reply BraveRewardsUI/Ads/AdsViewController.swift]: Lol its still not fixed 😂

**Comment by Brandon-T** [comment]
Hmm.. @kylehickinson I'm pretty sure I selected use spaces in Xcode and 2 spaces.. I just double checked.. Why it's happening I have ZERO idea.. but I had to edit it with TextEdit.. Also works if I edit it in Xcode 10.. but in 11 it completely screws up..

File: BraveRewardsUI/Ads/AdsViewController.swift (hunk 1 above)
Comment (Brandon-T) [reply BraveRewardsUI/Ads/AdsViewController.swift]: Fixed!

File: BraveRewardsUI/Ads/AdsNotificationHandler.swift [hunk 2]
```diff
@@ -47,6 +47,7 @@ public class AdsNotificationHandler: BraveAdsNotificationHandler {
     if adsViewController.parent == nil {
       presentingController.addChild(adsViewController)
       presentingController.view.addSubview(adsViewController.view)
+      adsViewController.didMove(toParent: presentingController)
```
Comment (kylehickinson) [inline BraveRewardsUI/Ads/AdsNotificationHandler.swift]: Missing setting up constraints here same as My First Ad

**Pull request merged**